java/src/main/java/ai/rapids/cudf/CompressionMetadataWriterOptions.java

```diff
@@ -124,5 +131,10 @@ public T withCompressionType(CompressionType compression) {
       this.compressionType = compression;
       return (T) this;
     }
+
+    public T withStripeSizeRows(int stripeSizeRows) {
+      this.stripeSizeRows = stripeSizeRows;
```

Comment:
Need some check on this "stripeSizeRows" input if there is any limition on this parameter, e.g.
`0< stripeSizeRows < (Some max value here)`.